main/styles.js

```diff
@@ -23,6 +23,7 @@ exports.getHighlightTheme = function (theme) {
 
   try {
     return fs.readFileSync(themePath, 'utf-8')
+      .replace(/([^;]+);/g, '$1 !important;')
```

Comment:
According to [spec](https://www.w3.org/TR/css-syntax-3/#consume-a-list-of-declarations) it seems that a semicolon is just a delimiter between CSS declarations, so this doesn't work in general for a last declaration in a block. And also semicolons may appear in comments (which is fine) and strings like font names or urls (probably not).

Maybe this transform wasn't really supposed to be generic but since it is applied to a Highlight.js stylesheet which can be either one of many built-in themes or even potentially user-defined (with `--highlight-stylesheet`), I'd say it's better to stay correct.

Have you tried other modules for that? For example, [postcss-safe-important](https://github.com/crimx/postcss-safe-important) seems really easy to integrate with:

```js
var postcss = require('postcss');
var important = require('postcss-safe-important');

postcss(important).process('foo { width: 100px; color: red }').css
//=> 'foo { width: 100px !important; color: red !important }'
```